src/raft/raw_node.rs

```diff
@@ -111,6 +111,10 @@ pub struct Ready {
     // If it contains a MsgSnap message, the application MUST report back to raft
     // when the snapshot has been received or has failed by calling ReportSnapshot.
     pub messages: Vec<Message>,
+
+    // MustSync indicates whether the HardState and Entries must be synchronously
+    // written to disk or if an asynchronous write is permissible.
+    pub must_sync: bool,
```

Comment:
we don't need this field, we will force syncing raft log all time.

Reply:
I wonder why we need this 'sync-log = true'?